samcli/lib/list/mapper_consumer_container.py

```diff
@@ -0,0 +1,9 @@
+"""
+Container for a mapper and a consumer
+"""
+
+
+class MapperConsumerContainer:
```

Comment:
What is this been used for?

Reply:
It's a container for each producer's mapper and consumer to be returned together from the factory

Reply:
I see, you can use `@dataclass` for this one to simplify the implementation